src/misc/unicode.cpp

```diff
@@ -519,8 +712,35 @@ static bool wide_to_dos(const std::vector<uint16_t>& str_in,
 		return true;
 	};
 
+	// Handle box drawing characters, if needed use specialized fallback
+	// strategy to guarantee table consistency
+	auto push_box_drawing = [&str_out](const map_grapheme_to_dos_t* mapping,
+	                                   const map_box_code_points_t* mapping_box,
+	                                   const Grapheme& grapheme) {
+		if (!mapping || !mapping_box) {
+			return false;
+		}
+
+		if (grapheme.HasMark()) {
+			return false; // not a box drawing character
+		}
+
+		const auto code_point = grapheme.GetCodePoint();
+		if (!mapping_box->count(code_point)) {
```

Comment:
Suggest quantity comparison (count == 0)

Reply:
OK, changed - everywhere in this file.